src/protocols/secure_channel/PASESession.h

```diff
@@ -272,6 +277,17 @@ class DLL_EXPORT PASESession : public Messaging::ExchangeDelegate, public Pairin
 
     Protocols::SecureChannel::MsgType mNextExpectedMsg = Protocols::SecureChannel::MsgType::PASE_Spake2pError;
 
+#ifdef CHIP_CSG_TEST_HARNESS //CSG_TRACE_BEGIN
+    std::map< std::string, std::map< std::string, std::string>> mPASETrace;
+    // PASE parameters maps
+    std::map<std::string,std::string> request_message_map;
+    std::map<std::string,std::string> response_message_map;
+    std::map<std::string,std::string> pake_1_message_map;
+    std::map<std::string,std::string> pake_2_message_map;
+    std::map<std::string,std::string> pake_3_message_map;
```

Comment:
I actually want to try and optimize these out of PASESession class entirely and make them global under our own file even the maps because 
- We won't have to worry about class lifecycle and destructors destroying these maps and strings.
- Reduces our footprint in conflict able code in sdk

Reply:
That would be cool

Reply:
https://github.com/chip-csg/connectedhomeip/issues/33